sklearnex/tests/test_n_jobs_support.py

```diff
@@ -73,6 +74,9 @@ def check_methods_decoration(estimator):
     # by default, [Nu]SVC.predict_proba is restricted by @available_if decorator
     if estimator_class in [SVC, NuSVC]:
         estimator_kwargs["probability"] = True
+    # by default, sklearn 1.5 PCA selects non-patched method
+    if estimator_class == PCA:
+        estimator_kwargs["svd_solver"] = "full"
```

Comment:
Ideally test_n_jobs_support should utilize sklearnex.tests._utils.SPECIAL_INSTANCES to make sure that the loss of a default check is covered by all of the test suite (and should be added there). This will save us headaches in the future.  Have we talked with @md-shafiul-alam about what we would need to do to support the default method?

Reply:
Btw, I don't think its necessary for this PR to do it, but something we should plan for.

Reply:
It should be separate discussion/PR.
`SPECIAL_INSTANCES` has a specific behaviour: it initializes estimators once and will reuse them in different places. While estimator API theoretically allows this without any troubles, usage of same estimator instance in different tests might cause unexpected behaviour in future.

Reply:
Thats absolutely true Maybe I should try to insert clone() at various places in the other tests using it.

Reply:
Please look at #1801, which should address that concern.